JavaScript/anagram.js

```diff
@@ -0,0 +1,30 @@
+// Given two strings, write a function to determine
+// if the second string is an anagram of the first.
+// Anagaram - Word/phrase/name formed by rearranging letters of another.
+// SAMPLE INPUT - anagram('anagram', 'nagaram') OUTPUT - true
+
+frequencyStr1 = {'a' : 3 ,'n' : 1, 'g' : 1, 'r' : 1, 'm' : 1}
```

Comment:
is this even a valid JS syntax?

i.e. shouldn't there be any of the `var`/`let`/`const` ??

Reply:
I guess this defines in global scope 🤔